#Valores de los componentes en x, y , d que describen los puntos en un obstaculo para la camara d435
float64 x
float64 y
float64 d


